scripts/install/config.default.sh

```diff
@@ -71,3 +71,10 @@ ADD_EPD_VISUALIZATION=false
 
 # The base URL (origin) of the SAP EPD Fiori launchpad
 EPD_VISUALIZATION_BASE_URL=
+
+#NPM connection info
+#NPM_URL must start by 'https://' and end with '/' char
+NPM_TOKEN=
+NPM_USER=
+NPM_URL= 
```

Comment:
no common.repositories url by default to avoid checkmarx warning. so user is forced to add it in config.sh